feat(docs-infra): update to @angular/* v7

IgorMinar opened the pull request.

IgorMinar committed:
feat(docs-infra): update to @angular/* v7

mary-poppins commented:
You can preview 0496ac2 at https://pr26555-0496ac2.ngbuilds.io/.

gkalpak commented on aio/package.json:
```diff
@@ -94,10 +94,10 @@
   },
   "devDependencies": {
     "@angular-devkit/build-angular": "~0.9.0-rc.1",
```
Why not this one too? :grin:

IgorMinar replied:
uh... good question. is this related to https://github.com/angular/angular-cli/issues/12624 ?

IgorMinar commented:
caretaker note: ci flake

IgorMinar committed:
fix(docs-infra): correct the version of @angular-devkit/build-angular

This dependency has not been correctly updated and I had to update it manually.

I think the issue is related to https://github.com/angular/angular-cli/issues/12624 which was fixed between CLI RCs and final.

mary-poppins commented:
You can preview 9631b3a at https://pr26555-9631b3a.ngbuilds.io/.

gkalpak commented on aio/package.json:
```diff
@@ -93,7 +93,7 @@
   },
   "devDependencies": {
-    "@angular-devkit/build-angular": "~0.9.0-rc.1",
+    "@angular-devkit/build-angular": "~0.10.0",
```
:+1:
